lib/routes/account.js

```diff
@@ -52,6 +52,7 @@ module.exports = function (
         var authSalt = crypto.randomBytes(32)
         var authPW = Buffer(form.authPW, 'hex')
         var locale = request.app.acceptLanguage
+        var userAgent = request.headers['user-agent']
```

Comment:
Same line, but unrelated comment...

Elsewhere `userAgent` refers to `lib/userAgent.js` so I had to do a double-take when I saw it used below in this file. Perhaps `userAgentString`?